src/components/pathway/tests/SearchPathwayCard.test.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { AppContext } from '@edx/frontend-platform/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import SearchPathwayCard from '../SearchPathwayCard';
+import { TEST_ENTERPRISE_SLUG } from './constants';
+
+import { renderWithRouter } from '../../../utils/tests';
+
+jest.mock('react-truncate', () => ({
+  __esModule: true,
+  default: ({ children }) => children,
+}));
+
+jest.mock('react-loading-skeleton', () => ({
+  __esModule: true,
+  // eslint-disable-next-line react/prop-types
+  default: (props = {}) => <div data-testid={props['data-testid']} />,
+}));
+
+const SearchPathwayCardWithAppContext = (props) => (
+  <AppContext.Provider
+    value={{
+      enterpriseConfig: { slug: TEST_ENTERPRISE_SLUG },
+    }}
+  >
+    <SearchPathwayCard {...props} />
+  </AppContext.Provider>
+);
+
+const TEST_PATHWAY_UUID = 'test-pathway-uuid';
+const TEST_TITLE = 'Test Title';
+const TEST_CARD_IMG_URL = 'http://fake.image';
+
+const defaultProps = {
+  hit: {
+    uuid: TEST_PATHWAY_UUID,
+    title: TEST_TITLE,
+    card_image_url: TEST_CARD_IMG_URL,
+  },
+};
+
+const propsForLoading = {
+  hit: {},
+  isLoading: true,
+};
+
+describe('<SearchPathwayCard />', () => {
+  test('renders the correct data', () => {
+    const { container } = renderWithRouter(<SearchPathwayCardWithAppContext {...defaultProps} />);
+
+    expect(screen.getByText(TEST_TITLE)).toBeInTheDocument();
+
+    expect(container.querySelector('.search-pathway-card > a')).toHaveAttribute(
+      'href',
+      `/#pathway-${TEST_PATHWAY_UUID}`,
+    );
+    expect(container.querySelector('.card-img-top')).toHaveAttribute('src', TEST_CARD_IMG_URL);
+  });
```

Comment:
can we add an expect to test if the skill tags are rendering correctly?

Reply:
Done